locale/en/blog/wg/diag-wg-update-2017-02.md

```diff
@@ -82,8 +64,21 @@ your experiences getting that info with tools and APIs old and new. [Open an
 issue][] in the Diagnostics WG for discussions or a PR in [nodejs/node][] to
 contribute code.
 
+Thank you!
+
+[async_hooks]: https://github.com/nodejs/node/pull/8531
+[CLI debugger]: https://nodejs.org/docs/v7.6.0/api/debugger.html
 [Debugging - Getting Started]: https://nodejs.org/en/docs/guides/debugging-getting-started/
-[Open an issue]: https://github.com/nodejs/diagnostics/issues/new
+[diag-agenda]: https://github.com/search?q=org%3Anodejs+label%3A%22diag-agenda%22&type=Issues
+[Diagnostics WG]: https://github.com/nodejs/diagnostics/issues
+[Inspector API]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
+[Inspector]: https://github.com/nodejs/node/issues?utf8=%E2%9C%93&q=label%3Ainspector%20
+[llnode]: https://github.com/nodejs/llnode
+[node-inspect]: https://github.com/nodejs/node-inspect
+[node-report]: https://github.com/nodejs/node-report
+[Node.js Foundation survey]: https://nodejs.org/static/documents/2016-survey-report.pdf
```

Comment:
This should be relative IMO `/static/documents/2016-survey-report.pdf`.

Reply:
Helped to change to a relative path.